instance/models/appserver.py

```diff
@@ -173,7 +173,8 @@ class AppServer(ValidateModelMixin, TimeStampedModel):
     name = models.CharField(max_length=250, blank=False)
     server = models.OneToOneField(OpenStackServer, on_delete=models.CASCADE, related_name='+')
     # The Instance that owns this. InstanceReference has related_name accessors like 'openedxappserver_set'
-    owner = models.ForeignKey(InstanceReference, on_delete=models.CASCADE, related_name='%(class)s_set')
+    owner = models.ForeignKey(InstanceReference, null=True, blank=True, on_delete=models.CASCADE,
```

Comment:
This change should be removed. The appserver's `owner` field is completely different from instance `owner` and should never be blank.